Remove sandbox holes related to the screenshot plugin

hadess opened the pull request:
The `org.gnome.Shell` "Flash" has been inaccessible from apps for a number of releases, and the screenshot plugin got updated to use a filechooser instead of requiring access to the Pictures directory.

bbhtt commented:
you have to update the submodule, this was fixed in https://github.com/flathub/shared-modules/commit/b0e640fd5065b15ccb66e1db00c7cc531004ae4c

hadess commented:
> you have to update the submodule, this was fixed in [flathub/shared-modules@b0e640f](https://github.com/flathub/shared-modules/commit/b0e640fd5065b15ccb66e1db00c7cc531004ae4c)

Yep, done that, thanks :)

hadess committed:
Remove D-Bus sandbox hole for screenshot plugin

The org.gnome.Shell "Flash" has been inaccessible from apps for a number
of releases, so remove access to it.

hadess committed:
Remove pictures dir sandbox holes for screenshot plugin

The screenshot plugin got updated to use a filechooser instead of
requiring access to the Pictures directory.
